nixos/modules/services/databases/redis.nix

```diff
@@ -25,202 +26,261 @@ in {
     (mkRemovedOptionModule [ "services" "redis" "appendOnlyFilename" ] "This option was never used.")
     (mkRemovedOptionModule [ "services" "redis" "pidFile" ] "This option was removed.")
     (mkRemovedOptionModule [ "services" "redis" "extraConfig" ] "Use services.redis.settings instead.")
```

Comment:
```suggestion
    (mkRemovedOptionModule [ "services" "redis" "extraConfig" ] "Use services.redis.*.settings instead.")
```

Reply:
Thanks, I forgot to change this old one. Note that it should be `services.redis.servers.*.settings` (there is a `servers` component because some options are global to all servers: `package` and `vmOverCommit`)